### PR TITLE
[ARTS-469] use main-latest tag and correct images names

### PR DESCRIPTION
**Pull request by eladiw**
## Description
Using correct images names (instead of placeholders) + changing to 'main-latest' which is now available

### Checklist:

- [X] Branch name follows convention (feature/arts-###-short-name OR fix/arts-###-short-name)
- [X] I have included a short-meaningful title, description and changes for this PR

### CODE REVIEW

File: sample_trial/definition.json [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
     "tenant_id": "5d23383f-2acb-448e-8353-4b4573b82276",
     "ui_client_id": "f352ce15-0142-4dfa-8e18-801ee6391557",
     "site_service": {
-        "image_name": "nginx",
+        "image_name": "ghcr.io/ndph-arts/trial-site-service",
         "image_tag": "latest"
     },
     "practitioner_service": {
-        "image_name": "nginx",
+        "image_name": "ghcr.io/ndph-arts/trial-practitioner-service",
```
Comment (tamirkamara) [inline sample_trial/definition.json]:
```suggestion
        "image_name": "ghcr.io/ndph-arts/practitioner-service",
```

File: sample_trial/definition.json [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
-    "id": "cpdemo",
+    "name": "trial0",
+    "id": "t01",
     "version": 1,
     "rootUser": "jondoe1@contoso.com",
     "tenant_id": "5d23383f-2acb-448e-8353-4b4573b82276",
     "ui_client_id": "f352ce15-0142-4dfa-8e18-801ee6391557",
     "site_service": {
-        "image_name": "nginx",
+        "image_name": "ghcr.io/ndph-arts/trial-site-service",
```
Comment (tamirkamara) [inline sample_trial/definition.json]:
```suggestion
        "image_name": "ghcr.io/ndph-arts/site-service",
```

**Review by tamirkamara — CHANGES_REQUESTED** [review]
FYI, this will fail until we start publishing latest images from the mts-services repo.

File: .github/workflows/validate.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name: ValidateTrialJsonSchema
 on:
   push:
-    branches: [ main, feature/** ]
+    branches: [ main, feature/**, fix/** ]
```
Comment (tamirkamara) [inline .github/workflows/validate.yml]: btw, I think this can be on all branches...

**Review by anatbal — DISMISSED** [review]

File: sample_trial/definition.json [hunk 4]
```diff
@@ -1,33 +1,37 @@
...
+        "image_name": "ghcr.io/ndph-arts/trial-practitioner-service",
         "image_tag": "latest"
     },
     "config_server_service": {
         "image_name": "ghcr.io/ndph-arts/trial-config-service",
-        "image_tag": "2d193042bc0766be588f04af705e6439e379acb8"
+        "image_tag": "latest"
+    },
+    "role_service": {
+        "image_name": "ghcr.io/ndph-arts/trial-role-service",
```
Comment (eladiw) [inline sample_trial/definition.json]:
```suggestion
        "image_name": "ghcr.io/ndph-arts/role-service",
```

File: .github/workflows/terraform-plan-apply-shared-rg.yml [hunk 5]
```diff
@@ -2,7 +2,7 @@ name: Terraform Plan and Apply for the Shared Resource Group
 
 on:
   push:
-    branches: [ main, feature/** ]
+    branches: [ main, feature/**, fix/** ]
```
Comment (eladiw) [inline .github/workflows/terraform-plan-apply-shared-rg.yml]:
```suggestion
    branches: [ ** ]
```

File: .github/workflows/terraform-plan-apply.yml [hunk 6]
```diff
@@ -2,7 +2,7 @@ name: Terraform Plan and Apply for Trials
 
 on:
   push:
-    branches: [ main, feature/** ]
+    branches: [ main, feature/**, fix/** ]
```
Comment (eladiw) [inline .github/workflows/terraform-plan-apply.yml]:
```suggestion
    branches: [ ** ]
```

File: .github/workflows/validate.yml (hunk 3 above)
Comment (eladiw) [inline .github/workflows/validate.yml]:
```suggestion
    branches: [ ** ]
```

**Review by anatbal — DISMISSED** [review]

File: .github/workflows/terraform-plan-apply-shared-rg.yml [hunk 7]
```diff
@@ -2,7 +2,7 @@ name: Terraform Plan and Apply for the Shared Resource Group
 
 on:
   push:
-    branches: [ main, feature/** ]
+    branches: [ '**' ]
```
Comment (tamirkamara) [inline .github/workflows/terraform-plan-apply-shared-rg.yml:5]: You can delete this line since it's "all"...

Comment (tamirkamara) [inline .github/workflows/terraform-plan-apply-shared-rg.yml]: you can delete this line to catch all branches.

**Review by tamirkamara — CHANGES_REQUESTED** [review]

**Review by tamirkamara — CHANGES_REQUESTED** [review]
It was decided that we'll use "main-latest" so need to update this PR.

**Review by tamirkamara — APPROVED** [review]

**Review by anatbal — APPROVED** [review]

**Pull request merged**